size_of_longest_substring.cpp

```diff
@@ -0,0 +1,138 @@
+/**
+ * @file
+ * @brief Length of longest substring without repeating characters
+ * @details
+ * From an input string, the length of the longest substring, in which the 
+ * substring is continuous and has no repeating characters, will be the output.
+ * https://leetcode.com/problems/longest-substring-without-repeating-characters/
+ * @author [Rafaela Ferreira](https://github.com/rafaela-eacf)
+ */
+ 
+#include <cassert>
+#include <iostream>
+#include <string>
+#include <algorithm>
```

Comment:
Please add a one-line description of what the library/header is for (**see the example below**).

```c++
#include <iostream>    /// for IO operations
#include <cassert>    /// for assert
```